cime_config/tests.py

```diff
@@ -197,12 +197,12 @@
         "time" : "02:00:00",
         "tests" : (
             # MMF tests
-            "SMS_D_Ln3_P96x1.ne4pg2_ne4pg2.F-MMF1",
-            "SMS_D_Ln3_P96x1.ne4pg2_ne4pg2.F-MMF2",
-            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX",
+            "SMS_Ln3_P96x1.ne4pg2_ne4pg2.F-MMF1",
```

Comment:
I realize these are not actually included in any regular testing, but why did you remove the debug builds? Debug often catches bugs we don't catch with higher optimization flags.

Reply:
I was having time limit issues, which I now realize because the script I was using was setting a 30 min time limit. I'll add the debug mode back for one of the SMS tests.